cosmwasm/deployment/terra2/tools/deploy.js

```diff
@@ -48,18 +47,6 @@ if (missing_artifacts.length) {
   process.exit(1);
 }
 
-const unexpected_artifacts = actual_artifacts.filter(
```

Comment:
why delete this code?

Reply:
I removed `global_accountant` from the artifacts list - there's no need to deploy it on terra2 devnet. Unless I remove this code, this script will always throw an error since it'll find the `global_account` in the artifacts folder but not in the artifacts list.

Reply:
Fixed this up